src/kata6.joinNames.js

```diff
@@ -1,3 +1,18 @@
-const joinNames = (namesObj) => {};
+const joinNames = (namesObj) => {
+   return namesObj.map(object => object.name).join(", ").replace(/,(?=[^,]+$)/, " &");
```

Comment:
Nice regex! Do you understand what it is doing? This would be a good example of when to use a comment, as at first glance it is not clear what is happening here